src/main/java/com/photolocos/enterprise/dao/IPhotoDAO.java

```diff
@@ -1,6 +1,6 @@
 package com.photolocos.enterprise.dao;
 
-import com.photolocos.enterprise.dto.PhotoDTO;
+import com.photolocos.enterprise.dto.Photo;
```

Comment:
Throughout the code I changed PhotoDTO to Photo as it is a cleaner naming convention for DTOs